.travis.yml

```diff
@@ -1,5 +1,16 @@
 sudo: false
 language: ruby
 rvm:
-  - 2.0.0
+- 2.0.0
```

Comment:
If I remember correctly, our first user had to upgrade to 2.2.6, or is this just the verison of rvm we use?